cluster/gce/gci/configure-helper.sh

```diff
@@ -356,6 +356,10 @@ function start-kubelet {
   fi
   echo "KUBELET_OPTS=\"${flags}\"" > /etc/default/kubelet
 
+  # Delete docker0 to avoid interference
+  ifconfig docker0 down
```

Comment:
nitpick: "ip link" is the preferred over ifconfig.

"ip link set docker0 down"